packages/@aws-cdk/toolkit/lib/actions/deploy/index.ts

```diff
@@ -189,6 +197,14 @@ export interface ExtendedDeployOptions extends DeployOptions {
    * @default - nothing extra is appended to the User-Agent header
    */
   readonly extraUserAgent?: string;
+
+  /**
+   * Allows adding CloudWatch log groups to the log monitor via
+   * cloudWatchLogMonitor.setLogGroups();
+   *
+   * @default - not monitoring CloudWatch logs
+   */
+  readonly cloudWatchLogMonitor?: CloudWatchLogEventMonitor;
```

Comment:
this is under `ExtendedDeployOptions` which is only used by `watch` and not used by customers. therefore i think it's fine/appropriate for it to be typed `CloudWatchLogEventMonitor` without the `I` because it is meant to always be the `CloudWatchLogEventMonitor` class.